galgebra/metric.py

```diff
@@ -78,6 +78,22 @@ def linear_expand(expr, mode=True):
     else:
         return list(zip(coefs, bases))
 
+def collect(A, nc_lst):
+    """
+    A is a linear combination of noncommutative symbols with scalar
+    expressions as coefficients.  collect() takes the terms containing
+    the noncommutative symbols in nc_list and sums them so no elements
+    of nc_list appear more than once in the sum.  collect() combines all
+    coefficients of a given element of nc_lst into a single coefficient.
```

Comment:
Since `sphinx.ext.napoleon` is part of our doc build, can this be
```
Parameters
------------
A :
    a linear combination of noncommutative symbols with scalar
    expressions as coefficients
nc_list :
    noncommutative symbols in A to combine

Returns
--------
A sum of the terms containing the noncommutative symbols in nc_list such that no elements
    of nc_list appear more than once in the sum. All
    coefficients of a given element of nc_lst are combined into a single coefficient.
```

Can this function get a unit test?

Reply:
Yes, that would be cool. I set up napoleon for exactly this purpose. But I haven't pay much attention to API doc correctness or doctests yet. It might be a seperate PR for all API doc, see some discussions on this matter in #43 .

Reply:
Committed your changes and it looks like this:

![image](https://user-images.githubusercontent.com/64258/67018417-cb692a00-f12d-11e9-9105-59bb03465f54.png)

Reply:
My guess is you have to indent the `A sum of` line, as
```
Returns
--------
    The description goes here, I don't know if it needs indenting
```

Reply:
Weird. I keep getting the following if I do so

![image](https://user-images.githubusercontent.com/64258/67020152-52b79d00-f130-11e9-9e06-5ffe2c954b6a.png)

Reply:
Perhaps try
```
sympy expression
    The description goes here, I don't know
    if it needs indenting